data/redirects/02040312-72c8-441e-a685-20e9333f3e1d.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "bookId": "02040312-72c8-441e-a685-20e9333f3e1d",
+    "pageId": "325e4afd-80b6-44dd-87b6-35aff4f40eac",
+    "pathname": "/books/introduction-sociology-2e/pages/preface"
```

Comment:
this is a mistake on the content side, we're going to have to also do a book version update to get the new content so this rename doesn't happen

Reply:
I should just re-run the script after https://github.com/openstax/rex-web/pull/1143 is merged, right?

Reply:
this pr is not merged yet, but I've checked that running a script again after this book is updated to `15.7` will result in not creating this file

Reply:
pr is merged, I've re-runed the script and file wasn't created